test/helpers/Relay.hh

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_GAZEBO_TEST_HELPERS_RELAY_HH_
+#define IGNITION_GAZEBO_TEST_HELPERS_RELAY_HH_
+
+#include <gtest/gtest.h>
+
+#include <ignition/gazebo/SystemLoader.hh>
+#include <ignition/gazebo/test_config.hh>
+
+#include "../plugins/MockSystem.hh"
+
+using namespace ignition;
+using namespace gazebo;
+
+namespace test
```

Comment:
This puts the class in `::test`. Wouldn't we want it in `ignition::gazebo::test`?

Reply:
Good call: f1b6451